projects/packages/import/src/endpoints/class-attachment.php

```diff
@@ -96,6 +96,26 @@ public function register_routes() {
 	 * @return WP_REST_Response|WP_Error Response object on success, WP_Error object on failure.
 	 */
 	public function create_item( $request ) {
+		$file_info  = $this->get_file_info( $request );
+		$attachment = $this->get_attachment_by_file_info( $file_info );
+		if ( $attachment ) {
+			$response = $this->prepare_attachment_for_response( $attachment, $request );
+
+			if ( \is_wp_error( $response ) ) {
+				return $response;
+			}
+
+			return new \WP_Error(
+				'attachment_exists',
+				__( 'The attachment already exists.', 'jetpack-import' ),
+				array(
+					'status'        => 409,
```

Comment:
I checked some `WP_Error` that the WordPress core code generates in the `WP_REST_Controller` family of classes.

When a `*_exists` event occurs the `status` generated is usually a simple `400`. See, for example, `WP_REST_Menus_Controller::create_item`, `WP_REST_Terms_Controller::create_item`, and others.

I agree that technically speaking it is, in fact, a "conflict" but maybe we should follow the style found in dotorg code? What do you think about this?